packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -1898,4 +1959,47 @@ export class TeamsAppSolution implements Solution {
       applicationIdUri: configResult.value.applicationIdUri,
     });
   }
+
+  private async getUserInfo(ctx: SolutionContext, email?: string): Promise<IUserList | undefined> {
+    const currentUser = await ctx.graphTokenProvider?.getJsonObject();
+    if (!currentUser) {
+      return undefined;
+    }
+    const tenantId = currentUser["tid"] as string;
+
+    if (email) {
+      const graphToken = await ctx.graphTokenProvider?.getAccessToken();
+      const instance = axios.create({
+        baseURL: "https://graph.microsoft.com/v1.0",
+      });
+      instance.defaults.headers.common["Authorization"] = `Bearer ${graphToken}`;
+      const res = await instance.get(`/users`);
```

Comment:
This request may only return first 100 users, you need to user filter to get user

Reply:
Fixed. Thx!